ansible/local.hosts

```diff
@@ -23,3 +23,10 @@ localhost ansible_connection=local #enabled_backends=lvm dock_endpoint=<host_ip>
 
 [worker-nodes]
 localhost ansible_connection=local
+
+[k8smaster]
```

Comment:
Add some comments that explain what is to be replaced and what this does.

Reply:
Done